OrbitClientGgp/ClientGgp.h

```diff
@@ -26,7 +26,7 @@ class ClientGgp final : public CaptureListener {
   bool InitClient();
   bool RequestStartCapture(ThreadPool* thread_pool);
   bool StopCapture();
-  void SaveCapture();
+  bool SaveCapture();
```

Comment:
Consider to return a ErrorMessageOr (since that is already done this way in capture_serializer::Save).

Reply:
I did but I realised the rest of methods called in ClientGgp (InitClient, RequestStartCapture and StopCapture) handle the errors themselves and then just return a boolean so I thought it would make sense to have the same kind of return for all

Reply:
then let's stick with that